monai/fl/utils/constants.py

```diff
@@ -33,11 +34,14 @@ class FlStatistics:
     #  https://github.com/Project-MONAI/MONAI/blob/df4a7d72e1d231b898f88d92cf981721c49ceaeb/monai/engines/trainer.py#L60
     TOTAL_EPOCHS = "total_epochs"
     TOTAL_ITERATIONS = "total_iterations"
+    NUM_EXECUTED_ITERATIONS = "num_executed_iterations"
 
 
 class BundleConst:
```

Comment:
Maybe better to call it `BundleKeys` or `RequiredBundleKeys`? Like the `WSIPatchKeys`, `FastMRIKeys`, `MetaKeys`, etc.
https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/enums.py#L409
And before running with a bundle, I would suggest to verify whether all the necessary keys are existing:
```py
for k in RequiredBundleKeys:
    if ConfigParser.get(k, None) is None:
        raise KeyError(...)
```

Thanks.